Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem "omnibus", github: ENV.fetch("OMNIBUS_GITHUB_REPO", "chef/omnibus"), branch: ENV.fetch("OMNIBUS_GITHUB_BRANCH", "main")
 gem "omnibus-software", github: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_REPO", "chef/omnibus-software"), branch: ENV.fetch("OMNIBUS_SOFTWARE_GITHUB_BRANCH", "main")
 gem 'artifactory'
+gem 'unf_ext', '0.0.8'
```

Comment:
We want the latest `'0.0.8'` which is `'0.0.8.2'` and we should also do pessimistic lock `'~> 0.0.8.2'`

Reply:
See diff for [v0.0.8 vs v0.0.8.2](https://github.com/knu/ruby-unf_ext/compare/v0.0.8...v0.0.8.2)